### ISSUE TITLE
[TF2] [Feature Request] Yearly Summer contract proposal

### ISSUE THREAD

**Issue by Arbitz** [ValveSoftware/Source-1-Games#4824]
This is a bit lengthy, though I feel this is something important for the longevity of TF2

**The outline**

Considering each year Halloween gets new contracts for the new maps, I feel it's unfair this system be restricted to Halloween. I feel a Summer contract system should be implemented for the new maps. It isn't (as far as I know) much of a hard ask due to the previously mentioned Halloween contracts that get added yearly.

**Reasons why I feel this is necessary**

- The lack of ways to earn Blood Money to use the contract shop feels annoying, being left with leftover blood money that's unusable feels unsatisfying and there's no way to get rid of it. The only way to even get more blood money is to play on an alternate account.

- Expanding from above, it gives players new goals and rewards players for skillfully playing the game. The most currently desired reward would be Blood money due to the issue mentioned above. Though the door is open for something new such as extra cases/gift bags or other oddities such as collectors kits.

- Extra substance to the updates and a "fuller feel".

These new summer contracts could be sat behind the Jungle Inferno pass, maybe renaming it to the "Mercenary Contract Pass" or something generic, and giving old contract owners a cool token.

**Comment by gaelcoral** [comment]
Please do this, there are gamemodes and maps that would benefit maps that nobody plays anymore like pass-time, Snowplow and so on.

**Comment by Wosheereal** [comment]
I wanted to expand on this by showing off a concept of one of the many ways the pass could look, I attempted to go for a TF logo with sun rays coming off of it, do note I am not a professional artist so this is going to look extremely amateur. 
https://imgur.com/a/2OqmeDe

**Comment by Wosheereal** [comment]
Also this pass has a good opportunity to add some Dragon Slayer based war paints from the community into the game thru the system either as keyless cases, contract rewards, or thru the blood money shop.

**Comment by Arbitz** [comment]
Yeah, its a good way to add certain UV skins and a way to get people to play underused map. And I feel the tradeoff between work the team needs to do and the amount of longevity, money and fun this'd bring... Its a wonder it hasn't been done
